### ISSUE TITLE
"PlaceAndBreakProtection" is insufficient

### ISSUE THREAD

**Issue by mastercake10** [Zrips/Jobs#886]
Currently, the plugin allows server owners to enable a so-called "Block protection" that seems to suppress job payments when a block gets placed or destroyed over and over again (see config comment for PlaceAndBreakProtection).

However, this option seems to also apply for ore-generators, thus users are unable to get their job payments while mining ores on generators.

I suggest implementing a new config option that only suppress payments from blocks that were exclusively placed by players beforehand.

This feature requests was raised from #885

**Comment by WarnDangerous** [comment]
it justs do not protect any blocks anymore :/

General:
# Enable blocks protection, like ore, from exploiting by placing and destroying same block again and again.
# Modify restrictedBlocks.yml for blocks you want to protect
PlaceAndBreakProtection: true
# For how long in days to keep block protection data in data base
# This will clean block data which ones have -1 as cooldown value
# Data base cleanup will be performed on each server startup
# This cant be more then 14 days
KeepDataFor: 1
# It allows you to pay for ore generator plugins continuously once you have broken the block in one place.
AllowBreakPaymentForOreGenerators: true

I take some blocks in my inventories, I place them, mine them, I earn exp
I tried to disable this feature, and now it's like before ( the AllowBreakPaymentForOreGenerators )
So the settings has a big problem, when I set

AllowBreakPaymentForOreGenerators: true

it's exactly like if I do

PlaceAndBreakProtection: false

**Comment by WarnDangerous** [comment]
Any news ?? It's very annoying :/

**Comment by montlikadani** [comment]
Its free to open a PR and implement this.

**Comment by nivcoo** [comment]
Hello, i use the last version and i can't add this in the config file, when i add it the line is clear :/ so now my players can't use oregen with jobs beacause by default is false

**Comment by mastercake10** [comment]
> Its free to open a PR and implement this.

Planned to open a PR for this, but it seems like there is a major issue with the so called "BlockProtection".
The newest Jobs version still gives money when for example placing iron ore and destroying it again. After repeating this over and over again for about 20 times, the plugin gives a penalty when breaking the block (negative experience and money). This even then applies to iron ores generated by a cobblestone generator, so your level successively decreases.
When you reload the plugin tho, it will remove this penalty and let you mine ores on this generator again, without giving penalties.

Assuming that this behavior is intended, I tried to remove the block from the BlockProtection list inside Jobs after they generate in the cobblestone generator, without success.

I tried to wrap my head around the code base, but I don't quite understand what exactly triggers the penalty. Someone needs to add java docs and cleanup the code to encourage others on getting their hands on the code...
